### ISSUE TITLE
Add herb run notification to time tracking plugin

### ISSUE THREAD

**Issue by HeelMePlz** [runelite/runelite#13175]
I noticed that there is a "birdhouse notification" option in the time tracking built-in plugin and thought a herb run notification could be very useful for others as well as myself. Typically, I do activities such as birdhouse runs only after a herb run, since herbs take a little longer to complete, so having the notification for herb runs instead of birdhouses would be much more useful to me.

**Comment by Hydrox6** [comment]
Duplicate of #2212

**Comment by HeelMePlz** [comment]
Ah, I didn't find it before because I typed herbs for my search sorry >.< Thank you for the reference link!
